table-api/src/main/java/io/deephaven/api/agg/Aggregation.java

```diff
@@ -0,0 +1,110 @@
+package io.deephaven.api.agg;
+
+import io.deephaven.api.ColumnName;
+import io.deephaven.api.JoinMatch;
+
+public interface Aggregation {
```

Comment:
Missing sorted first/last, abs-sum, and formula. Maybe something for "by-external", too.

Reply:
Looks a bit unfortunate, our existing sorted first/last does not allow specifying ascending or descending by column, like our pure sort does...

Reply:
Added AbsSum, Unique aggs. Similar to TableOperations though, I think it's useful as is, and hoping to merge shortly as-is. Pulling over additional aggs as necessary.